packages/cadl-rlc-emitter/test/integration/generated/models/readonlyProperties/README.md

```diff
@@ -0,0 +1,56 @@
+# ReadonlyProperties client library for JavaScript
```

Comment:
Please consider the comments here https://github.com/Azure/azure-sdk-for-js/pull/22044#discussion_r893496066

Reply:
updated